core-bundle/src/Resources/contao/dca/tl_search_term.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+$GLOBALS['TL_DCA']['tl_search_term'] = array
+(
+	// Config
+	'config' => array
+	(
+		'sql' => array
+		(
+			'keys' => array
+			(
+				'id' => 'primary',
+				'term' => 'unique'
+			)
+		)
+	),
+
+	// Fields
+	'fields' => array
+	(
+		'id' => array
+		(
+			'sql'                     => "int(10) unsigned NOT NULL auto_increment"
+		),
+		'term' => array
+		(
+			'sql'                     => "varchar(64) BINARY NOT NULL"
```

Comment:
This is not enough for words like "Grundstücksverkehrsgenehmigungszuständigkeitsübertragungsverordnung" or "Rinderkennzeichnungsfleischetikettierungsüberwachungsaufgabenübertragungsgesetz", but I guess that will not be a problem. 😄 

https://www.duden.de/sprachwissen/sprachratgeber/Die-langsten-Worter-im-Dudenkorpus

Reply:
The length is the same as before. The contao search does not support words longer than 64 characters, but I plan to remove that restiction when building the contao/search library.

Reply:
For what? There are only a handful of words exceeding the limit.

Reply:
There might be technical terms or something domain specific (like a special part number) which isn't part of a dictionary.

Reply:
I would like to remove the length restriction altogether. This would make it possible to reduce the storage length in the database even further. The current `varchar(64)` needs 256 bytes storage (because of `utf8mb4`), I would reduce that to a `binary(20)` column and store all words shorter than 20 utf-8 bytes directly and use a sha-1 hash for longer words. This would make the index ten times smaller and at the same time add support for insanely long words.

Reply:
Also if we can implement decomposition, words like provided in your example will be shortened very quickly ;)